retrieveProducts/src/service/productService.js

```diff
@@ -0,0 +1,73 @@
+import { getAllProducts, getProductById, createProduct } from '../repository/productsRepository';
+import { getStores } from '../repository/storeRepository';
+
+export const loadProducts = async () => {
+  try {
+    const [products, stores] = await Promise.all([getAllProducts(), getStores()]);
```

Comment:
Usually it's better to use `join`s in sql statements instead of combining data in the application

Reply:
Agree. will fix in scope of task5

Reply:
fix in the last commit